activesupport/lib/active_support/dependencies.rb

```diff
@@ -92,10 +92,17 @@ def initialize
         @stack = Hash.new { |h,k| h[k] = [] }
       end
 
+      # Calls the given block once for each element in stack, passing that
+      # element as a parameter. An Enumerator is returned if no block is given.
+      #
+      #   new_constants = ActiveSupport::Dependencies::WatchStack.new.new_constants
+      #   new_constants.each { |c| puts c }
+      #   # => RaisesNoMethodError
       def each(&block)
         @stack.each(&block)
       end
 
+      # Returns true if @watching stack is not empty
```

Comment:
This doesn't seem that useful to me too. Maybe it's because I have the code next to me that it doesn't make sense to add the proposed change.